packages/editor/src/components/post-layout-actions-panel/index.js

```diff
@@ -16,28 +9,28 @@ import { useCallback } from '@wordpress/element';
 /**
  * Internal dependencies
  */
-import { store as editPostStore } from '../../store';
+import EntitiesSavedStates from '../entities-saved-states';
+import PostPublishPanel from '../post-publish-panel';
+import PluginPrePublishPanel from '../plugin-pre-publish-panel';
+import PluginPostPublishPanel from '../plugin-post-publish-panel';
+import { store as editorStore } from '../../store';
 
 const { Fill, Slot } = createSlotFill( 'ActionsPanel' );
 
 export const ActionsPanelFill = Fill;
 
-export default function ActionsPanel( {
+export default function PostLayoutActionsPanel( {
 	setEntitiesSavedStatesCallback,
 	closeEntitiesSavedStates,
 	isEntitiesSavedStatesOpen,
+	hasActiveMetaboxes,
```

Comment:
This is not really a prop name that should exist in the editor package. The "metaboxes" are a edit-post only feature, what's the role of this prop?

Reply:
I renamed to `forceIsDirtyPublishPanel` because it's used for related checks in publish panel.